.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,19 @@
+- id: buf-breaking
+  name: buf breaking
+  language: golang
```

Comment:
Can you explain what the language key is for?

Reply:
this is what instructs pre-commit to use `go install` to set up the environment that executes the `buf` executable.

Reply:
Got it, thanks